3rdparty/libunwind/CMakeLists.txt

```diff
@@ -17,6 +17,24 @@ else()
   )
 endif()
 
+if (WIN32)
+  find_program(
+       BASH
+       NAMES bash
+       PATHS "C:/Program Files/Git/bin"
+       NO_DEFAULT_PATH
+   )
+   if (NOT BASH)
+      message(FATAL_ERROR "Git Bash not found! Please install Git in C:/Program Files/Git/bin")
```

Comment:
I don't think we need to hard-code it to this path.

Reply:
ok, will rectify